build.gradle

```diff
@@ -14,17 +14,18 @@ buildscript {
 
 plugins {
   id "com.jfrog.bintray" version "1.2"
-  id "us.kirchmeier.capsule" version "0.9.0"
+  //id "us.kirchmeier.capsule" version "1.0.2"
+//  id 'com.github.johnrengelman.shadow' version '7.1.2'
```

Comment:
Do we actually still need the Fat JAR? If not, all the capsule and shadow stuff can go.